crates/diesel_models/src/authentication.rs

```diff
@@ -90,7 +90,7 @@ pub enum AuthenticationUpdate {
         threeds_server_transaction_id: String,
         maximum_supported_3ds_version: common_utils::types::SemanticVersion,
         connector_authentication_id: String,
-        three_ds_method_data: String,
+        three_ds_method_data: Option<String>,
```

Comment:
Is this affecting any other flow?

Reply:
No. This field will only be returned by the authentication connector if 3ds method data collection is needed. We previously missed this.